src/utils/compose.js

```diff
@@ -0,0 +1,3 @@
+export default function(...funcs) {
+  return Component => funcs.reduce((comp, func) => func(comp), Component);
+}
```

Comment:
Jeg tror hvis vi deES6yfier litt så kan det bli bedre. Av og til synes jeg at for mye export default skaper litt forvirring fordi man mister litt nyttig navngivning.
`export default FormWithState = composeNewComponent(
  connect(mapStateToProps, { createWindow: Actions.createWindow }),
  withState(initialState)
)(Form);`
f.eks